### ISSUE TITLE
set Qt5_FOUND to FALSE

### ISSUE THREAD

**Issue by muezabdalla** [void-linux/void-packages#52912]
### Is this a new report?

Yes

### System Info

Void 6.11.5_1 x86_64 GenuineIntel uptodate FFF

### Package(s) Affected

paraview

### Expected behaviour

paraview compile complete

### Actual behaviour

I am trying to make a paraview template but it give this error:
```
=> paraview-5.13.1_1: installing target dependencies: base-devel-20181003_2 libXt-devel-1.3.0_1 qt5-tools-devel-5.15.11+20231124_5 python3-devel-3.12.7_1 python3-numpy-2.1.1_1 libopenmpi-5.0.4_1 qtchooser-66_3 qt5-qmake-5.15.11+20231124_5 qt5-plugin-odbc-5.15.11+20231124_5 qt5-plugin-mysql-5.15.11+20231124_5 qt5-plugin-pgsql-5.15.11+20231124_5 qt5-plugin-sqlite-5.15.11+20231124_5 qt5-plugin-tds-5.15.11+20231124_5 ...
=> paraview-5.13.1_1: running pre-configure hook: 00-gnu-configure-asneeded ...
=> paraview-5.13.1_1: running pre-configure hook: 01-override-config ...
=> paraview-5.13.1_1: running pre-configure hook: 02-script-wrapper ...
=> paraview-5.13.1_1: running do_configure ...
-- Using 'QtHelp' without 'QtWebEngine' will ignore embedded javascript and *.js files for documentation
-- Could not use git to determine source version, using version 5.13.1
-- Could not use git to determine source version, using version
-- VTK version: 9.3.20240617
CMake Warning (dev) at /usr/share/cmake-3.30/Modules/CMakeDependentOption.cmake:89 (message):
  Policy CMP0127 is not set: cmake_dependent_option() supports full Condition
  Syntax.  Run "cmake --help-policy CMP0127" for policy details.  Use the
  cmake_policy command to set the policy and suppress this warning.
Call Stack (most recent call first):
  VTK/Common/Core/CMakeLists.txt:79 (cmake_dependent_option)
This warning is for project developers.  Use -Wno-dev to suppress it.

CMake Warning at VTK/Rendering/OpenGL2/CMakeLists.txt:451 (message):
  X11::Xcursor not found; custom cursors will be ignored.


-- Could not use git to determine source version, using version 2.2.0
-- Could NOT find Qt5Svg (missing: Qt5Svg_DIR)
CMake Warning at VTK/CMake/vtkModule.cmake:5351 (find_package):
  Found package configuration file:

    /usr/lib64/cmake/Qt5/Qt5Config.cmake

  but it set Qt5_FOUND to FALSE so package "Qt5" is considered to be NOT
  FOUND.  Reason given by package:

  Failed to find Qt5 component "Svg" config file at
  "/usr/lib64/cmake/Qt5Svg/Qt5SvgConfig.cmake"



Call Stack (most recent call first):
  Qt/ApplicationComponents/CMakeLists.txt:329 (vtk_module_find_package)


CMake Error at VTK/CMake/vtkModule.cmake:5357 (message):
  Could not find the Qt5 external dependency.
Call Stack (most recent call first):
  Qt/ApplicationComponents/CMakeLists.txt:329 (vtk_module_find_package)


-- Configuring incomplete, errors occurred!
=> ERROR: paraview-5.13.1_1: do_configure: 'CFLAGS="-DNDEBUG ${CFLAGS/ -pipe / }" CXXFLAGS="-DNDEBUG ${CXXFLAGS/ -pipe / }" cmake ${cmake_args} ${configure_args} ${LIBS:+-DCMAKE_C_STANDARD_LIBRARIES="$LIBS"} ${LIBS:+-DCMAKE_CXX_STANDARD_LIBRARIES="$LIBS"} ${wrksrc}/${build_wrksrc}' exited with 1
=> ERROR:   in do_configure() at common/build-style/cmake.sh:77
```
I can not attach the template file so here is the content of it (it can be used as a starting point):
```bash
# Template file for 'paraview'
pkgname=paraview
version=5.13.1
revision=1
build_style=cmake
configure_args="-DCMAKE_BUILD_TYPE=Release"
hostmakedepends="cmake"
makedepends="base-devel libXt-devel qt5-tools-devel python3-devel python3-numpy libopenmpi qtchooser qt5-qmake qt5-plugin-odbc qt5-plugin-mysql qt5-plugin-pgsql qt5-plugin-sqlite qt5-plugin-tds"
short_desc="the world’s leading open source post-processing visualization engine."
maintainer="muez <muezabdalla777@gmail.com>"
license="BSD-3-Clause"
homepage="https://www.paraview.org/"
distfiles="https://www.paraview.org/paraview-downloads/download.php?submit=Download&version=v5.13&type=source&os=Sources&downloadFile=ParaView-v5.13.1.tar.gz"
checksum="2225de08ec3cb7ccf06519f7b7d488ba6cd1691a4d26608b003b077539d5ba16"
python_version=3
```

### Steps to reproduce

./xbps-src pkg paraview

**Comment by Johnnynator** [comment]
Codeblocks do start with ``` in their own line, the single ` is only for inline code. [Github Markdown Documentation](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#quoting-code)

The important part in the error message is this: ` Failed to find Qt5 component "Svg" config file at  "/usr/lib64/cmake/Qt5Svg/Qt5SvgConfig.cmake"` (Or in general all Svg mentions also in other lines).

In short you need `qt5-svg-devel`

**Comment by muezabdalla** [comment]
thanks for john zimmermann for fixing the problem by adding qt5-svg-devel to the makedepends variable

**Comment by muezabdalla** [comment]
Thank you for the fix
I will mention your name on github 

بتاريخ ٤ نوفمبر ٢٠٢٤ ٥:٢٦:١٧ م GMT+03:00، John Zimmermann ***@***.***> الرسالة:
